Dockerfile.composer

```diff
@@ -0,0 +1,16 @@
+FROM composer:1.6.3
+
+ARG composer_dev
```

Comment:
No problem with the argument names, but a slightly higher level of abstraction for the image name like [`build`](https://medium.com/@alexeiled/docker-pattern-the-build-container-b0d0e86ad601) and `build-dev` would be language-agnostic (or `php-build`|`php-build-dev` for a language-specific version). Other projects could easily require `node` or other specific tools to be installed in their own `build` image.

Later on when this pattern is replicated in `journal` (or even just the dummies), it rises the question on how to remove that duplication for ease of maintenance. There are several possible roads here...

Reply:
I think it'd be useful to isolate each build tool, Journal's going to need both Gulp and Composer.